event_routing_backends/settings/common.py

```diff
@@ -15,6 +15,7 @@ def plugin_settings(settings):
     settings.EVENT_ROUTING_BACKEND_COUNTDOWN = 30
     settings.EVENT_ROUTING_BACKEND_BULK_DOWNLOAD_MAX_RETRIES = 3
     settings.EVENT_ROUTING_BACKEND_BULK_DOWNLOAD_COUNTDOWN = 1
+    settings.EVENT_ROUTING_BATCH_SIZE = 5
```

Comment:
This doesn't seem to be used? I'm not seeing where batching is implemented.

Reply:
Not yet, the batching capabilities is still work in progress while we handle how to run a task to send the events in batch once every N seconds